sos/plugins/dnf.py

```diff
@@ -60,4 +66,17 @@ def setup(self):
         if self.get_option("history"):
             self.add_cmd_output("dnf history")
 
+        if self.get_option("history-info"):
+            history = self.call_ext_prog("dnf history")
```

Comment:
> Ah, I was partially blind.. Doesn't it makes sense to implicitly enable history option when history-info is enabled?

This is how we've handled previous cases (e.g. `-k rpm.rpmva` enables `--verify` just for the RPM module).

But.. I don't really like the proliferation of options here - experience shows that users don't tend to find them unless pointed to them explicitly (which somewhat removes the point of having them - if a 2nd touch is needed to re-request sos data with `-k dnf.history*` then we almost might just as well ask for `dnf history-whatever` itself..).

What are the actual costs for these operations, and can they not be made default (at least in the case of `dnf history`)?